### DISCUSSION TITLE
Idea

### DISCUSSION THREAD

**Discussion by Hitesh239** [thebarbican19/BatteryBoi#58] [Ideas]
This is nothing but wonderful to use. A humble request if you could add the batter minimum percentage (20%) and max percentage (80%) alert/alarm so User can set limit till what percentage they wish to charge mac.

**Comment by thebarbican19** [comment]
Hey @Hitesh239, This is actually in the works. Stay tuned my friend. In the meantime if you could review [BatteryBoi on Product Hunt](https://www.producthunt.com/products/batteryboi), I would appreciate it. Thanks again.
